### ISSUE TITLE
Multipart download:  is it really supported? I can't use it.

### ISSUE THREAD

**Issue by ibobak** [downthemall/downthemall#182]
**Desktop (please complete the following information):**
 - OS: Windows 7 x64
 - Browser and version: Firefox 70.0 (64bit)
 - DownThemAll! version: 4.2.4

**Describe the bug**
I used DownThemAll many years ago (still before it stopped working on the newest browsers).  Now I installed it again, but to my surprise it doesn't support multipart downloading at all. 

On the settings page
moz-extension://141633d1-e3d7-458c-b9d6-4f0cd3015830/windows/prefs.html
I see "network" tab, the settings are as on the attached screenshot below.

To demonstrate that this doesn't work, let us try to download, let us say, this: https://git-scm.com/download/win  - see the screenshot below.

**To Reproduce**
1. Go to https://git-scm.com/download/win
2. Click on 64-bit Git for Windows Setup, choose "DownThemAll" from popup menu, then "Save link with DownThemAll"
3. monitor the download progress.

**Expected behavior**
Must show that multiple parts are downloaded in parallel - as it was before.

**Screenshots**
![2019-10-30 10_51_46-New Issue · downthemall_downthemall](https://user-images.githubusercontent.com/8807043/67838940-81a41a80-fb03-11e9-944a-26543db7b8d9.png)
![2019-10-30 10_57_28-DownThemAll! Preferences](https://user-images.githubusercontent.com/8807043/67839187-1f97e500-fb04-11e9-8b55-035c6fd6760b.png)

**Comment by tonyhawz** [comment]
it is listed at https://www.downthemall.org/howto/features/
"Increase your download speed up to 400%"

**Comment by Anutrix** [comment]
@tonyhawz It was supported in old versions when it was not a WebExtension.

**Comment by drtdev** [comment]
> 
> 
> @tonyhawz It was supported in old versions when it was not a WebExtension.

maybe update the documentation? :) I just downloaded it because of that feature

**Comment by joedf** [comment]
@nmaier 
Perhaps it is no longer a limitation in WebExtensions?
This plugin claims to be capable of it:
https://addons.mozilla.org/en-CA/firefox/addon/multithreaded-download-manager/

**Comment by ivanzinhoa** [comment]
At least it resumes the download when the connection breaks down. It was my only one and favorite extension for Firefox in the old WinXP and early Win7 times.
